synapse/replication/tcp/streams/_base.py

```diff
@@ -231,8 +232,8 @@ class BackfillStream(Stream):
 
     def __init__(self, hs):
         store = hs.get_datastore()
-        self.current_token = store.get_current_backfill_token
-        self.update_function = store.get_all_new_backfill_event_rows
+        self.current_token = store.get_current_backfill_token  # type: ignore
+        self.update_function = store.get_all_new_backfill_event_rows  # type: ignore
```

Comment:
Whats the reason for the ignore here? Is the type unpredictable?

Reply:
:sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob:

Reply:
Basically mypy will not allow you to ever set a function like that, so we tell mypy to shut up about it. Its a bit naughty that we do it like this, but equally it works perfectly fine so I don't really fancy rewriting everything